test/test_pid_reuse.c

```diff
@@ -59,7 +59,7 @@ void* comp_file_multith (void *iterations)
 	/* Open the window. */
 	err = deflateInit(&c_stream, Z_DEFAULT_COMPRESSION);
 	if (err != 0) {
-		printf("nx_deflateInit err %d\n", err);
+		printf("deflateInit err %d\n", err);
```

Comment:
It's not a problem but all changes to test_pid_reuse could be in the same commit.